index.d.ts

```diff
@@ -204,3 +204,23 @@ export type RequireAtLeastOne<ObjectType, KeysType extends keyof ObjectType = ke
 	}[KeysType]
 	// …then, make intersection types by adding the remaining properties to each mapped type.
 	& Omit<ObjectType, KeysType>;
+
+/*
+Converts a readonly object into a mutable object. Inverse of Readonly<T>.
```

Comment:
```suggestion
Convert a readonly object into a mutable object. Inverse of `Readonly<T>`.
```

Reply:
I don't think we should say `readonly` as that might be confused with the `readonly` keyword, which cannot be used on arbitrary objects. Instead we should maybe say that the properties/keys are readonly. Thoughts?

Reply:
Agreed, I'll update the comment